src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs

```diff
@@ -530,12 +530,16 @@ private static void HookupHistory(ModelBuilder modelBuilder)
             .HasIndex(e => e.Epoch);
 
         modelBuilder
-            .Entity<KeyValueStoreEntryHistory>()
-            .HasIndex(e => new { e.KeyValueStoreEntityId, e.Key, e.FromStateVersion });
+            .Entity<KeyValueStoreEntryDefinition>()
+            .HasIndex(e => new { e.KeyValueStoreEntityId, e.FromStateVersion });
```

Comment:
I think we're missing one index for that query/filter that would include kv entity id, key, from_state_version

`WHERE d.key_value_store_entity_id = @keyValueStoreEntityId AND d.key = ANY(@keys) AND d.from_state_version <= @stateVersion"`

Reply:
I think we have all the necessary indices:
```
// used while iterating over KVStore entries
        modelBuilder
            .Entity<KeyValueStoreEntryDefinition>()
            .HasIndex(e => new { e.KeyValueStoreEntityId, e.FromStateVersion });

// used for key lookup
        modelBuilder
            .Entity<KeyValueStoreEntryDefinition>()
            .HasIndex(e => new { e.KeyValueStoreEntityId, e.Key });
```